packages/appeals-service-api/src/routes/v2/appeal-cases/controller.js

```diff
@@ -131,6 +149,17 @@ async function countByLpaCode(req, res) {
 	}
 }
 
+/**
+ * @param {any|undefined} bool
+ * @returns {boolean}
+ */
+function isValidBooleanString(bool) {
+	if (bool === undefined) {
+		return true;
```

Comment:
just checking, you definitley want this case to be true?

Reply:
Yes, could maybe do with a comment. Basically "optional boolean"!